lib/server/Server.js

```diff
@@ -19,6 +20,7 @@ var Server = {
         var onRouteHandled = config.onRouteHandled;
         var ServerAppToUse = config.ServerApp || ServerApp;
         var appRoot = environmentConfig.appRoot;
+        var proxyClientRequestsToApi = config.proxyClientRequestsToApi;
```

Comment:
I'm wondering if this should be the default behavior of Brisket since the Client App defaults to sending ajax requests to root relative urls. Would like to get input from others on this.